t/02-xsdynamic.t

```diff
@@ -14,6 +14,7 @@ use Test::More;
 plan skip_all => 'Dynaloading not enabled' if !$Config{usedl} or $Config{usedl} ne 'define';
 plan skip_all => "ExtUtils::CBuilder not installed or couldn't find a compiler"
   unless have_compiler();
+plan skip_all => 'Shared perl library' if $Config{useshrplib} eq 'true';
```

Comment:
This change looks unrelated to the rest of this change (and I suspect it's wrong)

Reply:
Yeah, I suspect this is hiding a problem

Reply:
I think too that this is hiding another problem